sdk/servicebus/Microsoft.Azure.ServiceBus/src/Primitives/AzureActiveDirectoryTokenProvider.cs

```diff
@@ -25,7 +20,7 @@ public class AzureActiveDirectoryTokenProvider : TokenProvider
         public AzureActiveDirectoryTokenProvider(AuthenticationCallback authenticationCallback, string authority, object state)
         {
             this.AuthCallback = authenticationCallback ?? throw Fx.Exception.ArgumentNull(nameof(authenticationCallback));
-            this.authority = authority ?? CommonAuthority;
+            this.authority = authority;
```

Comment:
Should we consider validating the `authority` as not null or empty?   Seems better to catch that up front then failing when trying to acquire a token later.